coordinator/src/services/validation/task-validator.ts

```diff
@@ -21,6 +21,15 @@ export function verifyTask(task: fhir.Task, scope: string): Array<fhir.Operation
   const statusSpecificErrors = performStatusSpecificValidation(task)
   validationErrors.push(...statusSpecificErrors)
 
+  if (task.requester) {
```

Comment:
One to check with Kevin, but I think requester means prescriber. Task messages (return & withdraw) will come from the dispenser, so this will never match. As far as I know, there's no place in the message for the dispenser details (yet).

Reply:
Actually the right field might be owner?